assets/xml/objects/object_bee.xml

```diff
@@ -1,19 +1,19 @@
 ﻿<Root>
     <File Name="object_bee" Segment="6">
-        <Animation Name="object_bee_Anim_00005C" Offset="0x5C" />
-        <DList Name="object_bee_DL_000550" Offset="0x550" />
-        <DList Name="object_bee_DL_000780" Offset="0x780" />
-        <DList Name="object_bee_DL_000848" Offset="0x848" />
-        <DList Name="object_bee_DL_000918" Offset="0x918" />
-        <DList Name="object_bee_DL_0009D0" Offset="0x9D0" />
+        <Animation Name="gBeeFlyingAnim" Offset="0x5C" />
+        <DList Name="gBeeBodyDL" Offset="0x550" />
+        <DList Name="gBeeHeadDL" Offset="0x780" />
+        <DList Name="gBeeAntennaeDL" Offset="0x848" />
+        <DList Name="gBeeWingLeftDL" Offset="0x918" />
+        <DList Name="gBeeWingRightDL" Offset="0x9D0" />
         <Texture Name="object_bee_TLUT_000A88" OutName="tlut_000A88" Format="rgba16" Width="4" Height="4" Offset="0xA88" />
         <Texture Name="object_bee_TLUT_000AA8" OutName="tlut_000AA8" Format="rgba16" Width="4" Height="4" Offset="0xAA8" />
         <Texture Name="object_bee_TLUT_000AC8" OutName="tlut_000AC8" Format="rgba16" Width="4" Height="4" Offset="0xAC8" />
         <Texture Name="object_bee_TLUT_000AE8" OutName="tlut_000AE8" Format="rgba16" Width="4" Height="4" Offset="0xAE8" />
```

Comment:
If possible we name the TLUTs after what they're for, there's a surprisingly large number here though... is it one per texture? Otherwise, just 1, 2, 3 is fine, see other files for the format.